src/daft-ir/src/proto/rel.rs

```diff
@@ -160,6 +160,12 @@ impl ToFromProto for ir::rel::LogicalPlan {
                 let udf_project = udf_project.to_proto()?.into();
                 proto::RelVariant::UdfProject(udf_project)
             }
+            Self::UrlDownload(url_download) => {
```

Comment:
Couldn't figure out how to serialize the IOConfig, so leaving as a followup. Other opts use the same macro